docs/docs/cmd/teams/meeting/meeting-create.mdx

```diff
@@ -0,0 +1,241 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams meeting create
+
+Create a new online meeting
+
+## Usage
+
+```sh
+m365 teams meeting create [options]
+```
+
+## Options
+
+```md definition-list
+`-s --startTime [startTime]`
+: The start time of the meeting. If not specified, the startTime will be set to the current time.
+
+`-e --endTime [endTime]`
+: The end time of the meeting. If not specified, the endTime will be set to one hour after the startTime.
+
+`--subject [subject]`
+: The subject of the meeting.
+
+`-p, --participants [participants]`
```

Comment:
I believe I advised to use `--participantUserNames` in the specs. It is long, but better, because we might add extra alternative options like `--participantUserIds`, in which case the `--participants` option will look out of place

Reply:
Done